Tasks/KubernetesManifestV0/src/actions/bake.ts

```diff
@@ -53,4 +53,48 @@ class HelmRenderEngine {
 
         return overrideValues;
     }
-}
\ No newline at end of file
+}
+
+interface NameValuePair {
```

Comment:
Can u please create a new file called helm-object-model in kubernetes manifest task and move the helm class in it? 

Secondly here also we will have to do the rework when we have second usage of Helm but since the current class has very small functionality, it should be fine.